### ISSUE TITLE
Error downloading com.dnanexus:wdltools_2.13:0.1.0-SNAPSHOT

### ISSUE THREAD

**Issue by mr-c** [dnanexus/dxWDL#422]
```
$ java --version
openjdk 11.0.7-ea 2020-04-14
OpenJDK Runtime Environment (build 11.0.7-ea+9-post-Debian-1)
OpenJDK 64-Bit Server VM (build 11.0.7-ea+9-post-Debian-1, mixed mode, sharing)
$ sbt compile
[info] Loading settings for project dxwdl-build from plugins.sbt ...
[info] Loading project definition from /home/michael/eclipse-workspace/dxWDL/project
[info] Loading settings for project root from build.sbt ...
[info] Set current project to dxWDL (in build file:/home/michael/eclipse-workspace/dxWDL/)
[info] Executing in batch mode. For better performance use sbt's shell
[info] Updating 
[info] Resolved  dependencies
[warn] 
[warn]  Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading com.dnanexus:wdltools_2.13:0.1.0-SNAPSHOT
[error]   Not found
[error]   Not found
[error]   not found: /home/michael/.ivy2/local/com.dnanexus/wdltools_2.13/0.1.0-SNAPSHOT/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/com/dnanexus/wdltools_2.13/0.1.0-SNAPSHOT/wdltools_2.13-0.1.0-SNAPSHOT.pom
[error]         at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:247)
[error]         at lmcoursier.CoursierDependencyResolution.$anonfun$update$35(CoursierDependencyResolution.scala:216)
[error]         at scala.util.Either$LeftProjection.map(Either.scala:573)
[error]         at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:216)
[error]         at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
[error]         at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:52)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:102)
[error]         at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:69)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:115)
[error]         at scala.util.control.Exception$Catch.apply(Exception.scala:228)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:115)
[error]         at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:96)
[error]         at sbt.util.Tracked$.$anonfun$inputChanged$1(Tracked.scala:150)
[error]         at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:129)
[error]         at sbt.Classpaths$.$anonfun$updateTask0$5(Defaults.scala:2950)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error]         at sbt.Execute.work(Execute.scala:290)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error]         at java.base/java.lang.Thread.run(Thread.java:834)
[error] (update) sbt.librarymanagement.ResolveException: Error downloading com.dnanexus:wdltools_2.13:0.1.0-SNAPSHOT
[error]   Not found
[error]   Not found
[error]   not found: /home/michael/.ivy2/local/com.dnanexus/wdltools_2.13/0.1.0-SNAPSHOT/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/com/dnanexus/wdltools_2.13/0.1.0-SNAPSHOT/wdltools_2.13-0.1.0-SNAPSHOT.pom
[error] Total time: 1 s, completed Jun 30, 2020, 12:31:42 PM
```

**Comment by commandlinegirl** [comment]
Hi @mr-c!

We have now included a [`wdlTools`](https://github.com/dnanexus-rnd/wdlTools) dependency in dxWDL that replaces Cromwell parser, type-checker, evaluator.  An example of how to set it is in our Github Actions workflow (a specific wdlTools commit needs to be checked out): https://github.com/dnanexus/dxWDL/blob/master/.github/workflows/scala.yml#L33

We'll soon be publishing it in Maven central so this step will not be necessary. I'll let you know here when it's done.

BTW: the master branch is still under testing and we released a [2.0.0-rc](https://github.com/dnanexus/dxWDL/releases/tag/v2.0.0-rc.) version of it here:  You can try it out if you'd like.

**Comment by mr-c** [comment]
Thank you @commandlinegirl !
